apps/web/server/routers/viewer.tsx

```diff
@@ -498,12 +498,13 @@ const loggedInViewerRouter = createProtectedRouter()
         There are connected calendars, but no destination calendar
         So create a default destination calendar with the first primary connected calendar
         */
-        const { integration = "", externalId = "" } = connectedCalendars[0].primary ?? {};
+        const { integration = "", externalId = "", credentialId } = connectedCalendars[0].primary ?? {};
```

Comment:
We can move all of this assignments to more rows.
```suggestion
        const [connectedCalendar] = connectedCalendards;
        const { primary: primaryCalendar } = connectedCalendar;
        const { integration = "", externalId = "", credentialId } = primaryCalendar;
```

Reply:
@alannnc that will fail if connecteCalendars is empty

Reply:
You can use fallbacks when destructuring.
<img width="277" alt="image" src="https://user-images.githubusercontent.com/4461358/179314320-2904f6f7-ede3-4f53-9418-61a7a7f8658f.png">